### PR TITLE
Integrating mapmerge2 checks into Travis

### PR DESCRIPTION
**Pull request by Sindorman**
Okay, this update is kinda big. Summary:

- Trimmed unused keys in Exodus telecomms

- Adds script that will run mapmerge2 on Travis to check branch for unused keys or key overflow, etc.

- Fixes matching indentation style in tag-matcher and converts it to use Python 3.6

- Converts mapmerge2 to be used by Python 3.4 and above. Instead of 3.6

- Removes Windows 1252 characters from Communication-blackout.dm that were not able to be seen in UTF-8 format.

Note: the last commit will fail because currently main level is broken

Example of no map issues:
2018-11-15_li

Example of issues:
2018-11-15_li 2

### CODE REVIEW

**Comment by Sindorman** [comment]
It only took me 4 hours? god, I thought it was 8 or something

**Comment by Sindorman** [comment]
I included map fix for mainlevel just to make Travis pass so we can merge it in. Provided by @BurgerLUA

**Comment by Sindorman** [comment]
Do not merge until #5612 is merged in!

File: tools/mapmerge2/travis_mapcheck.bat [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@echo on
+python travis_mapcheck.py
```
Comment (skull132) [inline tools/mapmerge2/travis_mapcheck.bat]: also, this script is useless, because you hardcoded the path in the mapcheck file.

Comment (Sindorman) [reply tools/mapmerge2/travis_mapcheck.bat]: I accidentally clicked resolved. Well no, the path never changes on travis. And it check for maps folder. I can make it use environment variables. How is it useless though ?

Comment (Sindorman) [reply tools/mapmerge2/travis_mapcheck.bat]: The script is intended to run **ONLY** on Travis, it is not designed to use Windows or for user to use it. The windows way of using it is to just run mapmerge process on a map.

This PR is just making mapmerge being run on Travis to check for map problems, it doesn't require to be adapted to windows

**Pull request merged**